server/store/kvstore/upgrade.go

```diff
@@ -22,14 +34,26 @@ func (s *Store) UpdateDatabase(pluginVersion string) error {
 		return s.System().SaveVersion(newestSchema.String())
 	}
 
-	// TODO: Uncomment following condition when version 1.1.0 is released
-	/*
-		currentSchemaVersion := semver.MustParse(v)
-		if err := s.UpgradeDatabaseToVersion11(currentSchemaVersion); err != nil {
+	for _, upgrade := range s.upgrades {
+		v, err := s.System().GetVersion()
```

Comment:
Do this need to call this on each iteration? Does it makes sense to update `v` after `SaveVersion` succeeds?

Reply:
Good catch :+1: I simplify the code to not fetch is on each iteration.